app/helpers/uploads_helper.rb

```diff
@@ -0,0 +1,35 @@
+require 'fileutils'
+require 'audit'
+require 'open3'
+
+module UploadsHelper
+  class Postprocessor
+    def self.process(f)
+      if self.respond_to?(File.extname(f)[1..-1])
```

Comment:
Mmm, good point.  I chose to do it this way so that in the future, we might enable plugins to contribute additional methods easily enough, and a hardcoded case/when statement isn't extensible.

Is there a way of having a dictionary of methods, instead?  I understand dictionaries; what I don't understand is how to write "method literals", so to speak...